screens/VitaminInfo.js

```diff
@@ -0,0 +1,86 @@
+import React, {Component} from "react";
+import {ScrollView, Text, StyleSheet, View} from "react-native";
+import YouTube from 'react-native-youtube';
+
+export default class VitaminInfo extends Component {
+    static navigationOptions = {
+        title: "Movement Vitamin Info"
+    }
+
+    constructor() {
+	super(...arguments);
+    }
+
+    render() {
+        const vitamin = this.props.navigation.state.params.vitamin;
+
+	return (
+            <ScrollView>
+              <View style={styles.title}>
+	        <Text style={styles.titleText}> {vitamin.name} </Text>
+              </View>
+              <View style={styles.videoAndDescTitle}>
+                <YouTube 
+                  apiKey={"AIzaSyCfgHrVK0IY8mb0C-URaV4WOBVaOnQtvyk"}
```

Comment:
Lets try to keep api keys out of the code base.

Reply:
We can run a fix for this once we have a server.